### PR TITLE
Translate latest news posts (it)

### PR DESCRIPTION
**Pull request by stomar**
Replaces #1265 with minor corrections.

### CODE REVIEW

**Comment by stomar** [comment]
@alepore, @kennyadsl, @mtylty

feedback?

File: it/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Ruby 2.3.0-preview2 è la seconda anteprima di Ruby 2.3.0.
+Sono state aggiunte molte nuove funzionalità e miglioramenti.
+
+E' stato aggiunto il [Frozen String Literal Pragma](https://bugs.ruby-lang.org/issues/11473).
+Su Ruby 2.1 `"str".freeze` è
+stato ottimizzato per ridurre il numero di oggetti allocati in memoria. Ruby 2.3
+introduce un nuovo commento 'magico' e un'opzione da linea di comando che
+permette di congelare tutte le stringhe nel codice sorgente.
+Inoltre per poter debuggare le proprie applicazioni, potete individuare dove sia
+stato creato l'oggeto che causa l'errore `"can't modify frozen String"` passando
```
Comment (kennyadsl) [inline it/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: oggeto -> oggetto

File: it/news/_posts/2015-12-16-unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+lang: it
+---
+
+E' stata trovata una vulnerabilità nella gestione delle stringhe in Fiddle e DL.
+A questa vulnerabilità è stato assegnato l'identificativo
+[CVE-2015-7551](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-7551).
+
+## Dettagli
+
+E' stato trovata una vulnerabilità nella gestion delle stringhe in Fiddle e DL.
```
Comment (kennyadsl) [inline it/news/_posts/2015-12-16-unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551.md]: gestion -> gestione

File: it/news/_posts/2015-12-25-ruby-2-3-0-released.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Questa è la prima release stabile della serie Ruby 2.3.
+Questa versione introduce nuove funzionalità, ad esempio:
+
+Il [frozen string literal pragma](https://bugs.ruby-lang.org/issues/11473).
+Su Ruby 2.1 `"str".freeze` è stato ottimizzato per ridurre il numero di oggetti allocati in memoria.
+Ruby 2.3 introduce un nuovo commento 'magico' e un'opzione da linea di comando che
+permette di congelare tutte le stringhe nel codice sorgente.
+Inoltre per poter debuggare le proprie applicazioni, potete individuare dove sia
+stato creato l'oggeto che causa l'errore `"can't modify frozen String"` passando
```
Comment (kennyadsl) [inline it/news/_posts/2015-12-25-ruby-2-3-0-released.md]: oggeto -> oggetto

File: it/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+semplice il debug delle nostre applicazioni.
+
+Sono stati aggiunti [RubyVM::InstructionSequence#to_binary e .load_from_binary](https://bugs.ruby-lang.org/issues/11788)
+come funzionalità sperimentali.
+Con queste funzionalità, possiamo creare un sistema di pre-compilazione ISeq (bytecode).
+
+Ruby 2.3 include molti fix per migliorare le performance.
+Ad esempio,
+[ottimizzazzione Proc#call](https://bugs.ruby-lang.org/issues/11569),
+[reconsidering method entry data structure](https://bugs.ruby-lang.org/issues/11278),
```
Comment (kennyadsl) [inline it/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: @simo2409 @stomar we should probably translate all link content here? or maybe none of them but it's proabably better to be more consistent.

File: it/news/_posts/2015-12-25-ruby-2-3-0-released.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+che mostra le possibili alternative quando si incontrano `NameError` o
+`NoMethodError` per rendere più semplice il debug delle nostre applicazioni.
+
+[RubyVM::InstructionSequence#to_binary e .load_from_binary](https://bugs.ruby-lang.org/issues/11788) sono stati aggiunte come funzionalità sperimentali.
+Con queste funzionalità, possiamo creare un sistema di pre-compilazione ISeq (bytecode).
+
+Questa release include anche molti miglioramenti alle performance, come ad esempio:
+[reconsidering method entry data structure](https://bugs.ruby-lang.org/issues/11278),
+[introducing new table data structure](https://bugs.ruby-lang.org/issues/11420),
+[ottimizzazione di Proc#call](https://bugs.ruby-lang.org/issues/11569),
```
Comment (kennyadsl) [inline it/news/_posts/2015-12-25-ruby-2-3-0-released.md]: same as [my previous comment](https://github.com/ruby/www.ruby-lang.org/pull/1396/files#r63863597)

**Comment by kennyadsl** [comment]
@simo2409 @stomar thanks!

**Comment by mtylty** [comment]
👍

**Comment by stomar** [comment]
Thanks @simo2409 @kennyadsl @mtylty 

Remaining fixes might be done later, I'd rather have this merged finally.

**Pull request merged**